content/collections/workflow/en/experiment-learnings.md

```diff
@@ -13,59 +13,113 @@ You’ve designed your experiment, rolled it out to your users, and given them e
 
 In the *Analysis* card, you’ll be able to tell at a glance whether your experiment has yielded **statistically-significant** results, as well as what those results actually are. Amplitude Experiment takes the information you gave it during the design and rollout phases and plugs them in for you automatically, so there’s no repetition of effort. It breaks the results out by variant, and provides you with a convenient, detailed tabular breakdown.
 
-{{partial:admonition type='note'}}
-This article continues directly from the [article in our Help Center on rolling out your experiment](/docs/feature-experiment/workflow/experiment-test). If you haven’t read that and followed the process it describes, do so before continuing here.
+Amplitude doesn't generate p-values or confidence intervals for experiments using binary metrics (for example, unique conversions) until each variant has 100 users **and** 25 conversions. Experiments using non-binary metrics need only to reach 100 users per variant.
+
+## Filter card
+
+On the Filter card, set criteria that updates the analysis on the page. Filter your experiment results with the following:
+
+* Date
+* Segment
+* Property
+
+### Date filter
+
+The date filter defaults to your experiment's start and end date. Adjust the range to scope experiment results to those specific dates.
+
+### Segment filter
+
+The segment filter enables you to select predefined segments, or create one ad-hoc. Predefined Experiment segments include:
+* All exposed users
+* Testers
+* Exclude testers
+* Exclude users who variant jumped
+
+These segments update in real-time.
+
+Click *+Create Segment* to open the Segment builder, where you can define a new segment on the fly. Segments you create in one experiment are available across all other experiments, and appear in the *All Saved Segments* category.
+
+### Property filter
+
+Filter your experiment results based on user or event properties. For example, create a filter that excludes users from a specific country or geographic region, or users that have a specific account type on your platform.
+
+## Data Quality card
+
+{{partial:admonition type="note" heading="Availability"}}
+Data Quality is available to organizations with access to Experiment who have recommendations enabled.
```

Comment:
Do we have a doc to link out to for how to enable the recommendations? Recommendations are enabled by default but people can disable them

Reply:
Is this experiment-specific recommendations? Or the recommendations in Audiences?

Reply:
<img width="434" alt="Screenshot 2025-01-30 at 5 05 20 PM" src="https://github.com/user-attachments/assets/913abfef-0e1b-47cf-aeb8-1ebbfa56c87b" />
experiment specific recommendations